kube/stateful_set.go

```diff
@@ -30,6 +30,12 @@ func NewStatefulSet(role *model.Role, settings ExportSettings) (helm.Node, helm.
 	spec := helm.NewMapping()
 	spec.Add("serviceName", fmt.Sprintf("%s-set", role.Name))
 	spec.Add("template", podTemplate)
+	// "updateStrategy" is new in kube 1.7, so we don't add anything to non-helm configs
+	// The default behaviour is "OnDelete"
```

Comment:
This feels like it belongs in a different PR (it's not directly related to secret generation)

Reply:
I agree; it was only merged in here because this integration branch was a substitute for `master` until we had restored backwards compatibility. Do you think this is important enough to split out into a separate PR. It is already in a separate commit: https://github.com/SUSE/fissile/pull/301/commits/4a2a4f866640c4d5d7d5d8fe96be86a4ad5d2b68, which points back to its own Trello card...

Reply:
😑 Alright, I guess.